src/shared/components/VideoPlayer/ControlsIndicator.style.ts

```diff
@@ -63,30 +81,33 @@ export const ControlsIndicatorTooltip = styled.div`
 const animationEasing = 'cubic-bezier(0, 0, 0.3, 1)'
 
 export const ControlsIndicatorTransitions = styled.div`
-  .indicator-enter {
+  .${INDICATOR_TRANSITION}-enter {
     ${() => StyledLoader} {
       opacity: 0;
       transform: scale(0.5);
     }
   }
 
-  .indicator-enter-active {
+  .${INDICATOR_TRANSITION}-enter-active {
     ${() => StyledLoader} {
       opacity: 1;
       transform: scale(1);
-      transition: opacity ${transitions.timings.sharp} ease-in, transform ${transitions.timings.sharp} ease-in;
+      transition: opacity 150ms ease-in, transform 150ms ease-in;
+      @media (hover: hover) {
+        transition: opacity 150ms 600ms ease-in, transform 150ms 600ms ease-in;
```

Comment:
Is this valid value for `transition`? The syntax is `property name | duration | easing function | delay`, what's that second `ms` value supposed to represent?

Reply:
Btw, maybe at this point it would be more clear to explicitly define specific properties, then you don't have to repeat the values inside the media query. For example:
```css
transition-property: opacity, transform;
transition-duration: 150ms;
transition-timing-function: ease-in;
@media (hover: hover) {
  transition-delay: 600ms;
}
```
(Not sure if this is exactly what you wanted to achieve but you get the point)